include/powsybl/iidm/DanglingLine.hpp

```diff
@@ -10,15 +10,87 @@
 
 #include <powsybl/iidm/CurrentLimitsAdder.hpp>
 #include <powsybl/iidm/Injection.hpp>
+#include <powsybl/iidm/ReactiveLimitsHolder.hpp>
+#include <powsybl/iidm/Validable.hpp>
+#include <powsybl/stdcxx/reference_wrapper.hpp>
 
 namespace powsybl {
 
 namespace iidm {
 
 class DanglingLine : public Injection {
+public:
+    class Generation : public Validable, public ReactiveLimitsHolder {
```

Comment:
GenerationImpl has methods `newMinMaxReactiveLimits` and `newReactiveCapabilityCurve` which are not used from implementation => seems to be useless in C++ context => not backported.